website/script.js

```diff
@@ -0,0 +1,116 @@
+function displayCurrentTimeEST() {
+    const currentTime = moment.tz('America/Toronto').format('hh:mm:ss A');
+    document.getElementById('currentTime').innerText = currentTime;
+}
+
+
+async function displayAllRoutes() {
+    // Fetches and processes the ferry schedule, adjusting dates to America/Toronto timezone
+    const response = await fetch('data/schedule.json');
+    const data = await response.json();
+    const schedules = data.schedules;
+    const today = moment.tz('America/Toronto').format('YYYY-MM-DD');
+    const routeSelect = document.getElementById('ferryRoute');
+
+    schedules.forEach(schedule => {
+        if (moment(schedule.start).isSameOrBefore(today) && (schedule.end === null || moment(schedule.end).isSameOrAfter(today))) {
```

Comment:
schedule.start and schedule.end will have to be updated to use the new model for start and end